_posts/2015-09-08-code-talks.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "ePages talks REST API"
+date: "2015-09-08 08:04:00"
+icon: code
+tags: misc
+categories: tech-stories conferences api
+authors: ["Birgit"]
+---
+
+Have you signed up for `code.talks` ?
```

Comment:
- code.talks 2015?
- should we link to the code talks website? 
- should we state that this conference is in Hamburg?